### ISSUE TITLE
When the conda environments are in the c:\users\{username}\.conda dir they do not work

### ISSUE THREAD

**Issue by rsheftel** [xlwings/xlwings#1187]
#### OS (e.g. Windows 10 or macOS Sierra)
Windows 10

#### Versions of xlwings, Excel and Python (e.g. 0.11.8, Office 365, Python 3.7)
Python 3.7.4
xlwings 0.16.0
Office 365
conda 4.7.12

#### Describe your issue (incl. Traceback!)
The standard single user installation setup for Anaconda and conda is to have the base Anaconda installation in C:\ProgramData\Anaconda3 and the user environments in c:\users\{username}\.conda\envs

This causes an issue because the setup for xlwings assumes that the environments are in the envs dir in the base installation. So when setting the "Conda Base:" to C:\ProgramData\Anaconda3 the starting of the UDF server will fail with the following:

![image](https://user-images.githubusercontent.com/11539619/69503326-ba5ac800-0ee6-11ea-9295-bea87ffeed31.png)

This is because the environments are in the users .conda/envs dir. But if the CONDA BASE is set to that location "C:\Users\{username}\.conda" the starting of the UDF server does not work because it cannot find the condabin directory, which is in the base installation location of C:\ProgramData\Anaconda3

![image](https://user-images.githubusercontent.com/11539619/69503350-f8f08280-0ee6-11ea-9d17-17be40cd96c4.png)

I would suggest two possible solutions, neither of which I can submit a PR for because I am not proficient in C/C++.. (1) add another input box for CONDA ENV DIR or (2) detect the "Conda Env" input box. if it is a directory string "c:/users/{username}/.conda/envs/myenv", set that for the environment using the full path if it is not a full path "myenv" then use the current method.

**Comment by fzumstein** [comment]
This should only be a documentation issue, please set the `interpreter` to the full path of the `pythonw.exe` in addition to the conda settings.

**Comment by rsheftel** [comment]
Thank you for that. When I update to the following:

Interpreter: C:\Users\rshef\.conda\envs\dev\python.exe
Conda Base:  C:\ProgramData\Anaconda3
Conda Env:  dev

I am able to start the UDF server, but when trying to use the code underneath I get the following:

> ImportError: Unable to import required dependencies:
> numpy: DLL load failed: The specified module could not be found.

Note that if I first open a command prompt and switch to my dev environment and then launch excel from the command line with "start excel", then it works properly. So it seems that it is not properly changing to the conda environment.

**Comment by fzumstein** [comment]
sorry, it should be analogous to how the conda commands work:

So make `Interpreter` blank again and put in the full path under `Conda Env`. As an example
```
Conda Base: C:\Users\felix\Miniconda
Conda Env: C:\Users\felix\Desktop\mylocation
```

**Comment by rsheftel** [comment]
That does not work. I set the following:

> Interpreter : {blank}
> Conda Base: C:\ProgramData\Anaconda3
> Conda Env: C:\Users\rshef\.conda\envs\dev

It causes the following:

![image](https://user-images.githubusercontent.com/11539619/69631754-3acf1500-101c-11ea-8cea-4a2be8c8e233.png)

Looks like it is trying to append the Conda Env to the Conda Base as a directory path.

**Comment by fzumstein** [comment]
did you restart the udf server after the changes?

**Comment by rsheftel** [comment]
Yes. I closed and relaunched Excel so it was a clean new launch.

**Comment by fzumstein** [comment]
ok, so seems there needs to be something fixed. Will try to look into it when I have some time.

**Comment by rsheftel** [comment]
I'm happy to help beta test any fixes if that is useful.

**Comment by FinancialRADDeveloper** [comment]
What is going on with this issue?   I don't seem to be able to use xlwings from VBA at all due to this bug.   I have tried to roll back to 0.15.8 to and it still doesnt work.

**Comment by fzumstein** [comment]
We prioritize issues following these rules: First come issues that impact xlwings PRO subscribers, then come issues that impact the most xlwings CE users.

**Comment by FinancialRADDeveloper** [comment]
Do you agree that there may be some issue here?   I don't mind working around it, maybe bu trying a non conda install of xlwings? 

I just wanted to check I hadn't done something bad on my install as I assume your software works for someone as mentioned in the examples.   I dont need a fix right now, I mainly need a way of running Python from VBA using xlwings that works

**Comment by fzumstein** [comment]
yes, as the issues states, if the conda envs are not in the default location, it currently doesn't work. Otherwise the settings for conda are described here: https://docs.xlwings.org/en/stable/addin.html#anaconda-miniconda

**Comment by FinancialRADDeveloper** [comment]
I am confused.... 

My conda *is* in the default location C:\Users\alan_\anaconda3

Then I have a v env called 

![image](https://user-images.githubusercontent.com/17732614/78661387-9cef2f00-78c6-11ea-9184-f26885d932ae.png)

So this issue says DON'T put them here... am I correct?   Your comment above seems to say it MUST be in here.. . whihh one is it?

**Comment by fzumstein** [comment]
This issue is about the `.conda` non-default dir. Yours (`envs`) is the default dir, so there must be something else wrong in your setup.

**Comment by fzumstein** [comment]
I believe this should be fixed by #2656
